search-app/src/cmr/search/site/routes.clj

```diff
@@ -23,6 +23,12 @@
         (GET "/"
              {context :request-context}
              (pages/home context))
+        (GET "/sitemap.xml"
+             {context :request-context}
+             (pages/sitemap-master context))
+        (GET "/site/sitemap.xml"
```

Comment:
Why do we have this top level XML file in addition to the sitemap master XML file?